lib/ex_unit/lib/ex_unit/doc_test.ex

```diff
@@ -167,12 +167,26 @@ defmodule ExUnit.DocTest do
     * `:import` — when `true`, one can test a function defined in the module
       without referring to the module name. However, this is not feasible when
       there is a clash with a module like Kernel. In these cases, `import`
-      should be set to `false` and a full `M.f` construct should be used.
+      should be set to `false` and a full `M.f` construct should be used, or
+      the `:setup` option can be used to fine-tune the configuration.
+
+    * `:setup` — takes a quoted expression of the configuration desired
+      to run for each test.
```

Comment:
Can we change the wording a bit here? We are not configuring anything, rather we are setting up the environment for the doctests to run with the help of the quoted expression.